src/routes/backpack/newscrud.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+|--------------------------------------------------------------------------
+| Backpack\NewsCRUD Routes
+|--------------------------------------------------------------------------
+|
+| This file is where you may define or change all of the routes that are
```

Comment:
True. But this is the file we intend to publish later on, with a command, so I thought the description should be broad enough. Confusing instead of broad?